task07-consume-real-data/index.html

```diff
@@ -27,128 +27,7 @@
-        }
-      }
-
-      let characters = [];
-      let islands = [];
-      let mistObjects = [];
-
-      async function populateCharacters(cantCharacters) {
-        let elem = document.getElementById("charactersContainer");
-        let components = "";
-        for (let pos = 0; pos < cantCharacters; pos++) {
-          const character = characters[pos];
-          components += `<a class="character__card" href="">
-          <span class="character__card__name">${character.name}</span>
-          <div class="character__card__gradient"></div>
-          <img src="./images/${character.img}" />
-        </a>`;
-        }
-        elem.innerHTML = components;
-      }
-
-      async function populateIslands(cantIslands) {
-        let elem = document.getElementById("islandsContainer");
-        let components = "";
-        for (let pos = 0; pos < cantIslands; pos++) {
-          const island = islands[pos];
-          components += `<a href="" class="island__card">
-          <div class="island__card__textContainer">
-            <span class="island__card__name">${island.name}</span>
-            <div class="island__card__locationContainer">
-              <span class="material-icons"> place </span>
-              <span class="island__card__location">${island.Location}</span>
-            </div>
-            <span class="material-icons" name="islandLike"> favorite </span>
-          </div>
-          <img
-            src="./images/${island.img}"
-          />
-        </a>`;
-        }
-        elem.innerHTML = components;
-      }
-
-      async function populateMistObjects(cantMistObjects) {
-        let elem = document.getElementById("mistObjectsContainer");
-        let components = "";
-        for (let pos = 0; pos < cantMistObjects; pos++) {
-          const mistObject = mistObjects[pos];
-          components += `<a href="" class="mistObject__card">
-          <img
-            src="./images/${mistObject.img}"
-
-          />
-          <div class="mistObject__card__textContainer">
-            <h3 class="mistObject__card__name">${mistObject.name}</h3>
-            <p class="mistObject__card__description">
-              ${mistObject.description}
-            </p>
-          </div>
-        </a>`;
-        }
-        elem.innerHTML = components;
-      }
-
-      function sortData(a, b) {
-        return a.name > b.name ? 1 : -1;
-      }
-
-      function loadData() {
-        fetch("https://static.akurey.com/trainings/OnePieceInformation.json")
-          .then((response) => response.json())
-          .then((data) => {
-            islands = data.islands.sort(sortData);
-            characters = data.characters.sort(sortData);
-            mistObjects = data.mysticObjects.sort(sortData);
-
-            populateCharacters(4);
-            populateIslands(3);
-            populateMistObjects(4);
-          });
-      }
-
-      window.onload = loadData;
-    </script>
+    <script src="./script.js"></script>
```

Comment:
move script to bottom of file, we add css to the head but scripts to the bottom so they dont affect render times